Core/clim-core/presentations.lisp

```diff
@@ -1310,6 +1310,7 @@ and used to ensure that presentation-translators-caches are up to date.")
               ,@body)))
         (t
          `(lambda (,(car args) &key ,@(cdr args) &allow-other-keys)
+            (declare (ignorable ,(car args)))
             ,@body))))
```

Comment:
maybe, but if somebody's going through the trouble of specifying the keyword args, they probably should use them. I'd leave this as is for now.